lib/easybill-rest-client/api_client.rb

```diff
@@ -26,55 +29,58 @@ def request_collection(method, endpoint, params = {})
     end
 
     def request(method, endpoint, params = {})
-      Retryable.retryable(
-        tries: tries,
-        sleep: retry_cool_off_time,
-        on: EasybillRestClient::TooManyRequests
-      ) do
-        response = perform_request(method, endpoint, params)
-        process_response(response)
+      @request_logger = RequestLogger.new(logger: logger, request_id: Time.now.to_f)
+      retry_on(EasybillRestClient::TooManyRequests) do
+        retry_on(Net::OpenTimeout) do
```

Comment:
Could you make retries optional? I'd like to have a choice to catch an exception and proceed without waiting.